jekyll-feed.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "nokogiri", "~> 1.6"
   spec.add_development_dependency "rake", "~> 12.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "rss" if RUBY_VERSION >= '3.0.0'
```

Comment:
This change is not correct.
A gemspec is evaluated only during `gem build` calls or via a Gemfile pointing to the gem's repository.

In other words, if I were to cut the gem release using Ruby 2.7, then `rss` won't be marked as a dependency.

Reply:
Isn't that the whole point since `rss`gem is only required since Ruby 3.0?

Reply:
I thought `rss` only needed for tests, in `spec/spec_helper.rb`. At least I could not find any other place where it is required. So the idea was: If on Ruby 3, add `rss` so tests can run. `if` can be removed, I was just unsure if there is some constraint preventing from things working on older Ruby versions.

Reply:
> I thought `rss` is only needed for tests, in `spec/spec_helper.rb`

Yes, that is correct.
To maintain consistency when cutting the release on differing Ruby versions, I recommend moving the `rss` gem declaration to the Gemfile.